### ISSUE TITLE
Bug on player displays on the Dynamap

### ISSUE THREAD

**Issue by Zarika92** [Hebbinkpro/PocketMap#20]
Since I installed your plugin and put all the recommended virions and they are well launched, after 1 hour of waiting to move a little everywhere on my server, no indication of player has appeared, do you have an idea or is it a bug? Thank you! (use Pocketmap 0.4.3)

**Comment by Hebbinkpro** [comment]
Hi thanks for reporting the bug,

I was already aware of the bug, but haven't found the time to finish the update for the plugin yet.

But this is an issue in the web files, so you can fix it yourself if you want to.
To fix the issue, go to the file `plugin_data/PocketMap/web/static/script/main.js`. Find the following function
```js
function updateMarker(player) { 
     let pos = player["pos"]; 
     let latLng = L.latLng(-pos.z, pos.x);
     ...
}
```
Change `let latLng = L.latLng(-pos.z, pos.x);` 
to `let latLng = L.latLng(-pos.z/16, pos.x/16);`
After this it will behave properly.

**Comment by Hebbinkpro** [comment]
Fixed in v0.4.4
